src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -63,6 +64,75 @@ private static String getFileName(String uri) {
+                }
+                return "http://" + address + "/conf";
+            }
+        }
+        private Protos.CommandInfo getCommandInfo() {
+            MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
+            CommandInfo.Builder commandInfo = CommandInfo.newBuilder();
+            if (myriadExecutorConfiguration.getNodeManagerUri().isPresent()) {
+            /*
+            Overall this is messier than I'd like due to Mesos 1760. We can't let mesos untar the distribution,
+            since it will change the permissions.  Instead we simply download the tarball and execute tar -xvpf.
+            We also pull the config from the resource manager and put them in the conf dir.  This is also why we need
+            frameWorkSuperUser
+            */
+
+                LOGGER.info("Using remote distribution");
+
+                String executorCmdPrefix = "export CAPSULE_CACHE_DIR=`pwd`;echo $CAPSULE_CACHE_DIR; ";
+                if (cfg.getFrameworkUser().isPresent()) {
+                    executorCmdPrefix += "sudo -E -u " + cfg.getFrameworkUser().get() + " -H ";
+                }
+                executorCmdPrefix += "java -Dcapsule.log=verbose -jar ";
+
+                String nmURI = myriadExecutorConfiguration.getNodeManagerUri().get();
+
+                //todo(DarinJ) support other compression, as this is a temp fix for Mesos 1760 may not get to it.
+                String tarCmd = "sudo tar -zxpf " + getFileName(nmURI);
+
+                String chownCmd = "";
+                if (cfg.getFrameworkUser().isPresent()) {
+                    chownCmd += "sudo chown " + cfg.getFrameworkUser().get() + " .";
+                }
+
+                String configCopyCmd = "cp conf " + cfg.getYarnEnvironment().get("YARN_HOME") + "/etc/hadoop/yarn-site.xml";
+
+                String cmd = tarCmd + "&&" + configCopyCmd + "&&" + chownCmd + "&&" + executorCmdPrefix + myriadExecutorConfiguration.getPath();
+
+                //get configs directly from resource manager
+                URI configURI = URI.newBuilder().setValue(getConfigurationUrl()).build();
+                //We're going to extract ourselves, so setExtract is false
+                URI executorURI = URI.newBuilder().setValue(nmURI).setExecutable(true).build();
+                commandInfo.addUris(configURI).addUris(executorURI).setUser(cfg.getFrameworkSuperUser().or(""))
+                        .setValue("echo \"" + cmd + "\";" + cmd);
+            } else {
+                String cmdPrefix = " export CAPSULE_CACHE_DIR=`pwd` ;" +
+                        "echo $CAPSULE_CACHE_DIR; java -Dcapsule.log=verbose -jar ";
+                String executorPath = myriadExecutorConfiguration.getPath();
+                String cmd = cmdPrefix + getFileName(executorPath);
+                URI executorURI = URI.newBuilder().setValue(executorPath)
+                        .setExecutable(true).build();
```

Comment:
I think standard style is to put .build()'s on their own line. At least that's how I see it in the hdfs framework and most of the rest of this one.